README.md

```diff
@@ -585,6 +585,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
   * [deployhq.com](https://www.deployhq.com/) — 1 project with ten daily deployments (30 build minutes/month)
   * [drone](https://cloud.drone.io/) - Drone Cloud enables developers to run Continuous Delivery pipelines across multiple architectures - including x86 and Arm (both 32-bit and 64-bit) - all in one place
   * [LayerCI](https://layerci.com) — CI for full stack projects. One full stack preview environment with 5GB memory & 3 CPUs.
+  * [pre-commit.ci](https://pre-commit.ci) - CI service for the [pre-commit](https://pre-commit.com) framework will always be free for open source repositories.
```

Comment:
I think its a bit too single tech focussed for a generic list, so I'll pass on this one thanks